id-authentication-default.properties

```diff
@@ -645,7 +650,7 @@ mosip.ida.vci.supported.cred.types=VerifiableCredential,MOSIPVerifiableCredentia
 
 # Regex to validate handles with provided key as the postfix
 # if the input handle is +855345353453@phone then the provided regex is used to validate the input.  
-mosip.ida.handle-types.regex={ '@phone' : '^\\+91[1-9][0-9]{7,9}@phone$' }
+mosip.ida.handle-types.regex={ '@phone' : '^\\+855[1-9][0-9]{7,9}@phone$' }
```

Comment:
This is camdgc specific property value, Pleaqse revert it back to previous value.

Reply:
Reverted.